addons/web/static/src/views/fields/percentage/percentage_field.js

```diff
@@ -8,10 +8,13 @@ import { useInputField } from "../input_field_hook";
 import { useNumpadDecimal } from "../numpad_decimal_hook";
 import { standardFieldProps } from "../standard_field_props";
 
-import { Component } from "@odoo/owl";
+import { Component, onWillUpdateProps, useRef, useState, useExternalListener } from "@odoo/owl";
```

Comment:
Hello @kcv-odoo 👋🏼 
Thanks for you work. But this looks, at a first sight, quite a heavy change for a layout issue (all the percentage field thing I mean).
I tried by just using w-25 on a 16.0 runbot and it seems enough. What did I miss?
I am also wondering whether w-25 is a real good solution. Did you test in mobile?
Thanks!

Reply:
OR maybe has it been fixed in the meanwhile?